.github/ISSUE_TEMPLATE/proposal--lightning-talk.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Talk proposal - Lightning Talk (~10 min)
+about: "Propose a short talk that is about 10 minutes in length "
+title: "Talk proposal: <Talk name>"
+labels: Lightning
+assignees: ""
+---
+
+# Title of talk
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid the content like product pitches or vendor driven talk. 
```

Comment:
```suggestion
A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid content like product pitches or vendor-driven talks. 
```